### ISSUE TITLE
List filters in auto-generated documentation

### ISSUE THREAD

**Issue by dbrgn** [encode/django-rest-framework#6608]
## Checklist

- [ ] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](https://www.django-rest-framework.org/community/third-party-packages/#about-third-party-packages) where possible.)

## Description

Feature request: The built-in auto-generated docs (`rest_framework.documentation.include_docs_urls`) should include a list of fields that can be filtered on. This list should match the filterable fields when clicking the "Filter" button in the DRF admin.

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — I think this one can be closed as the feature it targets no longer exists in DRF.

This request asks for filter-field listing inside the auto-generated docs surfaced by `rest_framework.documentation.include_docs_urls`. That entire coreapi/coreschema-based docs stack was deprecated for several releases and finally removed in DRF 3.17.0.

Evidence:

- PR [#9895](https://github.com/encode/django-rest-framework/pull/9895) "Drop deprecated coreapi support" merged 2026-03-16, deleting `rest_framework/documentation.py` (including `include_docs_urls`) and the coreapi schema hooks in `rest_framework/filters.py`.
- [`docs/community/release-notes.md`](https://github.com/encode/django-rest-framework/blob/main/docs/community/release-notes.md) lists "Drop deprecated coreapi support" under **Breaking changes** for 3.17.0 (released 2026-03-18).
- On current `main`, `rest_framework/documentation.py` is gone; only historical mentions remain in the 3.6 announcement and the release-notes archive.

So the surface the original request was about is no longer there. Anyone wanting filter-aware API docs today would reach for OpenAPI-based tooling (e.g. `drf-spectacular`, which renders `DjangoFilterBackend` filters as parameters out of the box).

If I'm missing a coreapi-equivalent path that still survives somewhere, happy to dig further — otherwise this looks closeable.

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the PR/commit references and the filesystem checks above were verified manually against current `main`.*

**Comment by browniebroke** [comment]
Good catch, thanks for bringing this up. I verified the claim and can confirm it's been deleted!
